src/printing.py

```diff
@@ -12,6 +12,14 @@
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
 
+print("x is %i, y is %f, z is %s" % (x, y, z))
```

Comment:
The problem is asking you to round y to the second decimal. Check out this link for the first two https://mkaz.blog/code/python-string-format-cookbook/ and this one for the third https://stackoverflow.com/questions/45310254/fixed-digits-after-decimal-with-f-strings